jbpm-designer-backend/src/main/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnit.java

```diff
@@ -591,6 +598,25 @@ public void evaluateWorkDefinitions(Map<String, WorkDefinitionImpl> workDefiniti
                 } else {
                     workDefinition.setCustomEditor(null);
                 }
+                
+                // Get URLs so that we can create a URL-based ClassLoader, if necessary
```

Comment:
Could we please externalize this into a method that can be called if there is an enum type paramValue ?